playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -10,6 +10,9 @@
   remote_user: root
```

Comment:
cc @mtnbikenc @ingvagabund ... I know adhoc isn't a place where we are directly targeting **but** wanted to loop you in just in case we kill this playbook as it has multiple items for porting.